gym_madras.py

```diff
@@ -0,0 +1,160 @@
+"""
```

Comment:
Can we improve the description here. Any developer who uses this code would hugely depend on the comments leave in this file.

Reply:
I agree with you. I wrote something trivial. You suggest what more can be written to help developers out.

Reply:
IMO: 
->Gym Wrapper for Madras environment
-> Built on top of gym_torcs: list the enhancements in made in gym_madras from gym_torcs.